langroid/embedding_models/models.py

```diff
@@ -32,6 +34,21 @@ class SentenceTransformerEmbeddingsConfig(EmbeddingModelsConfig):
     devices: Optional[list[str]] = None
 
 
+class FastEmbedEmbeddingsConfig(EmbeddingModelsConfig):
+    """Config for qdrant/fastembed embeddings,
+    see here: https://github.com/qdrant/fastembed
+    """
+
+    model_type: str = "fastembed"
+    model_name: str = "BAAI/bge-small-en-v1.5"
+    dims: int = 384
```

Comment:
`dims` should be removed. It should all work without this.

Reply:
Missed that. Fixed [`3a5cdfb` (#521)](https://github.com/langroid/langroid/pull/521/commits/3a5cdfb5eb924707402cc1f67d9084b21d68d721).